FrontEnd/src/components/adminPage/AdminConstants.js

```diff
@@ -0,0 +1,2 @@
+export const DEFAULT_PAGE_SIZE = 10;
```

Comment:
Let's call this file just `constants` it's a more common approach

Reply:
done